From e70dc094265c0418bbd895d9657611ac8d509a1c Mon Sep 17 00:00:00 2001
Message-Id: <e70dc094265c0418bbd895d9657611ac8d509a1c.1649937132.git.geliang.tang@suse.com>
From: Yang Li <yang.lee@linux.alibaba.com>
Date: Wed, 22 Dec 2021 09:55:13 +0800
Subject: [PATCH] NTB/msi: Fix ntbm_msi_request_threaded_irq() kernel-doc
 comment
Git-commit: e70dc094265c0418bbd895d9657611ac8d509a1c
Patch-mainline: v5.17-rc1
References: bsn#170
Modified-by-SEL: No


Add the description of @msi_desc and change the @devname to @name
in ntbm_msi_request_threaded_irq() kernel-doc comment to remove
some warnings found by running scripts/kernel-doc, which is caused
by using 'make W=1'.
drivers/ntb/msi.c:285: warning: Function parameter or member 'name' not
described in 'ntbm_msi_request_threaded_irq'
drivers/ntb/msi.c:285: warning: Function parameter or member 'msi_desc'
not described in 'ntbm_msi_request_threaded_irq'
drivers/ntb/msi.c:285: warning: Excess function parameter 'devname'
description in 'ntbm_msi_request_threaded_irq'

Reported-by: Abaci Robot <abaci@linux.alibaba.com>
Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
Signed-off-by: Jon Mason <jdmason@kudzu.us>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/ntb/msi.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/ntb/msi.c b/drivers/ntb/msi.c
index 3f05cfbc73af..60953cf54af6 100644
--- a/drivers/ntb/msi.c
+++ b/drivers/ntb/msi.c
@@ -260,8 +260,9 @@ static int ntbm_msi_setup_callback(struct ntb_dev *ntb, struct msi_desc *entry,
  * @handler:	Function to be called when the IRQ occurs
  * @thread_fn:  Function to be called in a threaded interrupt context. NULL
  *              for clients which handle everything in @handler
- * @devname:    An ascii name for the claiming device, dev_name(dev) if NULL
+ * @name:    An ascii name for the claiming device, dev_name(dev) if NULL
  * @dev_id:     A cookie passed back to the handler function
+ * @msi_desc:	MSI descriptor data which triggers the interrupt
  *
  * This function assigns an interrupt handler to an unused
  * MSI interrupt and returns the descriptor used to trigger
-- 
2.34.1

